src/components/AuthContainer.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import VersionLabel from './VersionLabel';
+import dots from './../assets/authDots.png';
+
+import '../stylesheets/auth.scss';
+
+export default function AuthContainer({children}) {
+  return (
+    <>
+      <div className="auth__version">
```

Comment:
This layout should be changed before applied to other screens. The question is: when?